src/+otp/Problem.m

```diff
@@ -1,67 +1,95 @@
 classdef (Abstract) Problem < handle
     
     properties (SetAccess = private)
-        Name % A human-readable representation of the problem
+        % A human-readable representation of the problem
+        Name %MATLAB ONLY: (1,:) char
     end
     
     properties (SetAccess = protected)
-        RHS
+        RHS %MATLAB ONLY: (1,1) otp.RHS = otp.RHS(@() [])
     end
     
     properties (Access = private)
-        Settings
-        ExpectedNumVars
+        ExpectedNumVars %MATLAB ONLY: {mustBeScalarOrEmpty, mustBeInteger, mustBePositive}
+        
+        % Determines if the Problem properties are still being set. Once set, it
+        % is ok to call onSettingsChanged.
+        Initialized = false
```

Comment:
Oh, so now we won't call onsettingschanged three times when making the problem?

Reply:
Exactly. This was true before this PR but handled differently